apis/management.cattle.io/v3/rke_types.go

```diff
@@ -333,6 +335,9 @@ type KubeletService struct {
 type KubeproxyService struct {
 	// Base service properties
 	BaseService `yaml:",inline" json:",inline"`
+
+	// Enabled
+	Enabled *bool `yaml:"enabled" json:"enabled,omitempty" norman:"default=false"`
```

Comment:
Default should probably be true as to not break existing deployments.

Reply:
You're right, I even have it defaulting to `true` in RKE: https://github.com/rancher/rke/pull/1831/files#diff-ee322344e47d033c1509dc709149f86eR84